specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/examples/RedisEnterpriseSkuList.json

```diff
@@ -0,0 +1,608 @@
+{
+    "parameters": {
+      "api-version": "2022-01-01",
+      "subscriptionId": "subid",
+      "location": "westus"
+    },
+    "responses": {
+      "200": {
+        "body": {
+            "value": [
+              {
+                "resourceType": "redisEnterprise",
+                "locationInfo": {
+                  "location": "West US",
+                  "zoneMappings": [
+                    {
+                      "logicalZone": "1",
+                      "physicalZone": "uswest-AZ01"
+                    },
+                    {
+                      "logicalZone": "2",
+                      "physicalZone": "uswest-AZ02"
+                    }
+                  ],
+                  "locationCapabilities": [
+                    {
+                      "name": "ZoneAware",
+                      "value": true
+                    },
+                    {
+                      "name": "FDsAvailable",
```

Comment:
Would there be any value in exposing the number of zones/fault domains, instead of just a true/false boolean?

Reply:
Eben actually returning the actual list of logical zones that this customer subscription supports, could make a lot of sense -- in case it's not just [1, 2] -- I don't think we can make that API work though until we've had the subscription info registered/cached with the RP.